olive/cli/base.py

```diff
@@ -235,19 +235,18 @@ def update_input_model_options(args, config):
 
 def add_logging_options(sub_parser: ArgumentParser):
     """Add logging options to the sub_parser."""
-    log_group = sub_parser.add_argument_group("logging options")
-    log_group.add_argument(
+    sub_parser.add_argument(
         "--log_level",
         type=int,
         default=3,
         help="Logging level. Default is 3. level 0: DEBUG, 1: INFO, 2: WARNING, 3: ERROR, 4: CRITICAL",
     )
-    return log_group
+    return sub_parser
 
 
 def add_remote_options(sub_parser: ArgumentParser):
     """Add remote options to the sub_parser."""
-    remote_group = sub_parser.add_argument_group("remote options")
+    remote_group = sub_parser
```

Comment:
Any reason we want to remove group here? Does it display differently on the doc?

Reply:
The group titles are not helping to clearly and quickly identify which options belong to one command.